Misc/NEWS.d/next/Library/2023-12-29-17-57-45.gh-issue-113569.qcRCEI.rst

```diff
@@ -0,0 +1,2 @@
+Display list of actual calls in unittest `Mock.assert_has_calls` failure
```

Comment:
It is a reStructuredText, not Markdown.

```rst
Display if there were no actual calls in unittest ``Mock.assert_has_calls`` failure.
```
or
```rst
Display if there were no actual calls in unittest :meth:`~unittest.mock.Mock.assert_has_calls` failure.
```

Reply:
Fixed in https://github.com/python/cpython/pull/113573/commits/f0ebf498e6869642524efeac0827d1771d71355f